README.md

```diff
@@ -65,6 +65,12 @@ Gets or sets whether the drop down is enabled. If you want to apply a specific s
 * **accessoryViewVisible** - *boolean* (Default: true)  
 Gets/sets whether there will be an accessory view (toolbar with Done button) under iOS. Valid only when running on iOS.
 
+* **itemsTextAlignment** - *String*
+Gets or sets the alignment for items in the DropDown. eg: `itemsTextAlignment="center"`.
+
+* **itemsPadding** - *String* android only! for now
```

Comment:
I prefer to not have properties supporting only one of the OS-es. So let's keep this PR for the `itemsTextAlignment` and when you get `itemsPadding` working for iOS you can submit another one with just that?